### DISCUSSION TITLE
mapillary - whose fault, mine or OsmAnd?

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#20029] [General]
I added a picture - Image key 821219429935270
`https://www.openstreetmap.org/node/10249048140`

There is no image in the application.

**Comment by ThomasMeister123** [comment]
Are there images for any other POIs in OSMAnd?

**Reply by ZeljkoBG** [reply]
I don't know. But in the application of POI there is an option 'Online photos'.

**Reply by DmitryAlexei** [reply]
- please update your map to the latest version and enable live updates 
- you also need to enable Mapillary plugin 

For more information see docs:
https://osmand.net/docs/user/personal/maps#osmand-live
https://osmand.net/docs/user/plugins/mapillary

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/1f429a40-94ba-4169-9192-77e3f2c71301"  width="300" height="auto"/> </td>
  </tr>
</table>
